docs/bipoc.rst

```diff
@@ -0,0 +1,74 @@
+Our #bipoc channel
+==================
+
+This is a space dedicated to supporting BIPOC members within our community.
```

Comment:
Perhaps we could explain what "BIPOC" means here? I know, but not everyone who runs into this page will know.

Reply:
I agree! One of my friends raised the same point. We'll add an explanation here or somewhere close to the top of the page.

Reply:
```suggestion
This is a space dedicated to supporting BIPOC (black, Indigenous and people of color) members within our community.
```